### PR TITLE
Plans for Q2

### PR DESCRIPTION
**Pull request by karthikv2k**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/245)

### CODE REVIEW

**Comment by karthikv2k** [comment]
#88

File: roadmap.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Q2 2019
+1. Distirbuted trianing TF (1.x and 2.0)
+1. Distirbuted training for LightGBM
+1. TF-Serving as an option for delpolyment in addition to Seldon model server
+1. Improve documentation and testing
+
+# Q3 and Q4 2019 Big Rocks
+1. Support for distributed training in other frameworks: PyTorch, Spark ML, etc.
+1. Integrations: KF-Serving, KF-Pipelines, Katib
+1. Integration tests for on-prem use case
```
Comment (abhi-g) [inline roadmap.md]: Can we add another item for better support for AppendBuilder?

Comment (karthikv2k) [reply roadmap.md]: done.

**Comment by abhi-g** [comment]
fixes #88

**Pull request merged**